engines/query-sparql-file/package.json

```diff
@@ -172,10 +171,12 @@
     "@comunica/mediator-join-coefficients-fixed": "^2.6.8",
     "@comunica/mediator-number": "^2.6.8",
     "@comunica/mediator-race": "^2.6.8",
-    "@comunica/runner": "^2.6.8",
```

Comment:
I think we'll also need this one for the dynamic engine from config to work.

Reply:
This dependency is also never mentioned throughout the whole dir.

Reply:
Like the config one, this is needed during dep injection, so we'll also need a workaround for this.